implementations/python/sugar-lift-python-source/src/sugar_lift_python_source/lifter.py

```
@@ -2245,24 +2307,40 @@ def _literal_default(node: ast.expr) -> Json:
         )
     if isinstance(node, ast.Dict):
         keys = [
-            none_const() if key is None else _literal_default(key)
+            none_const() if key is None else _literal_default(key, tentative_expr)
             for key in node.keys
         ]
-        values = [_literal_default(value) for value in node.values]
+        values = [_literal_default(value, tentative_expr) for value in node.values]
         entries = [
             ctor("python:dict_entry", key, value)
             for key, value in zip(keys, values)
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)